test/controllers/actions/AnalyseMessageActionSpec.scala

```diff
@@ -59,7 +59,8 @@ class AnalyseMessageActionSpec extends AnyFreeSpec with Matchers with MockitoSug
       val application = GuiceApplicationBuilder()
         .overrides(bind[MessageAnalyser].toInstance(messageAnalyser))
         .configure(
-          "metrics.jvm" -> false
+          "metrics.jvm" -> false,
+          "microservice.services.features.nctsMonitoringEnabled" -> true
```

Comment:
Can you explain why this is required? It's set in the config so I'd expect the unit tests to pick it up